README.md

```diff
@@ -226,6 +231,13 @@ WHERE rds.storage_encrypted = false
 return a.name as AWSAccount, count(rds) as UnencryptedInstances
 ```
 
+### What users have the Grammarly for Chrome extension installed?
```

Comment:
I think it's beneficial to provide some queries around analyzing risks. This would also help in the PR review to see if the schema makes it easy.

Reply:
gave example of querying by risk_total

Reply:
[blocker] Let's not make a judgment on extension but focus on the risk score provided by the data.

Reply:
Personally I wouldn't block on adding risk analysis documentation.  I think a better place for that would be a blog post.

Reply:
No judgement was made on risk of a specific extension, the first example only gives an example of querying by name. The second example is querying all extensions by risk score using some sample criteria from the risk scoring page. Changed name in sample to a fake name to clear up any potential concern though.